src/TodoApp/components/TodoList/TodoListComponent.jsx

```diff
@@ -0,0 +1,58 @@
+import styled from 'styled-components';
+
+const Input = styled.input`
+	top: 0;
+	left: 0;
+	height: 15px;
+	width: 25px;
+	background-color: #0bb89b !important;
```

Comment:
Generally you should avoid using `!important`, there's usually a better way to override the styles.

Btw, CSS specificity is a very important concept to deeply understand, and it is more complex than it seems at first: https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity